pages/checkout/terms.tsx

```diff
@@ -6,10 +6,16 @@ import { FormattedMessage, useIntl } from "react-intl"
 import { useDispatch, useSelector } from "react-redux"
 import styled from "styled-components"
 
+import { Banner } from "../../components/Banner"
 import { CheckoutWizard } from "../../components/CheckoutWizard/CheckoutWizard"
 import { Container } from "../../components/Container"
 import { Dispatch, RootState } from "../../store"
 
+const Title = styled.div`
+  display: flex;
+  column-gap: 0.5rem;
```

Comment:
What do we need display and column gap for here? Should use the Typography component anyway :)